packages/ui/src/hooks/useInternalRelayChains.ts

```diff
@@ -0,0 +1,19 @@
+import { useRelayChains } from '@reservoir0x/relay-kit-hooks'
+import { useRelayClient } from './index.js'
+
+const DEFAULT_CACHE_OPTIONS = {
+  staleTime: 1000 * 60 * 5, // 5 minutes
+  gcTime: 1000 * 60 * 5,
+  refetchOnMount: false,
+  refetchOnWindowFocus: false
+} as const
+
+export const useInternalRelayChains = () => {
+  const relayClient = useRelayClient()
+
+  return useRelayChains(
```

Comment:
Why use this over just using `useRelayChains` ? If the host app calls `useRelayChains` you can reuse the data.

Reply:
This is just a wrapper hook to make it so we don't have to pass in the same query options in every place we use the hook. It's using the same query key as `useRelayChains`, so the data can be shared with the host app as long as the options are the same

Reply:
nice